src/stan_inference.jl

```diff
@@ -47,7 +47,7 @@ function generate_theta(n,priors)
 end
 
 function stan_inference(prob::DiffEqBase.DEProblem,t,data,priors = nothing,
-                            stanmodel = nothing;alg=:rk45,
+                            stanmodel = nothing;alg=:adams,
```

Comment:
```suggestion
                            stanmodel = nothing;alg=:rk45,
```

I wouldn't make it the default because if the rk45 is any good it should perform better on a lot of standard non-stiff problems.

Reply:
Yep, sounds good!